src/main/kotlin/mate/academy/model/IsoscelesTrapezoid.kt

```diff
@@ -0,0 +1,14 @@
+package mate.academy.model
+
+private const val MULTIPLIER = 0.5
+
+class IsoscelesTrapezoid(val color: String, val baseSide: Double, val height: Double): Figure(color) {
+    override fun getArea(): Double {
+        return MULTIPLIER * baseSide * height
+    }
+
+    override fun draw() {
+        println("Figure: isosceles trapezoid, area: ${getArea()}.sq units, base side: " +
```

Comment:
FYI you may use triple quote mark to avoid concatenation.
for example """some string ....
                        bla-bla-bla """.someBuiltInExtensionFun()